packages/gatsby-theme-docs/src/layouts/release-notes-detail.js

```diff
@@ -50,15 +50,21 @@ const LayoutReleaseNotesDetail = (props) => {
           <LayoutPageHeader>
             <SpacingsInline alignItems="center">
               <AngleLeftIcon size="medium" color="primary" />
-              <Link href="/releases" noUnderline={true}>
```

Comment:
Why did this change?

Reply:
I needed the back button functionality/behavior of the browser to preserve the filters state when users use this link to return to release notes list, and we don't have back button functionality in the link component.

Reply:
@davifantasia  it doesn't work as simple as replacing the link with a back navigation trigger.  If you open the details page URL in a fresh tab the link points into nothing / the user's empty tab page. 

Since the link is named "All Release Notes" it should stay as is and navigate to `/releases`.     

Since as far as I know the browser API does not allow to read the previous history state a "smart" link would be hacky, work only inside a gatsbyJS site etc.  -> I don't think it's worth trying a "smarter" link.

Reply:
@nkuehn I can attempt to go in a little bit of a different direction - pass the filter states to details page. And when "All Releases" is clicked, then the state can be passed back to the list page. Would that work?

Reply:
I would suggest to leave this behavior for a follow up. Just keep the link as-is for now.

Reply:
I also suggest to leave the "All" link to point to all.   Giving a page that does not have state URL state would just incur more follow-up issues, e.g. with search crawlers etc.  

Don't mess with the basic specs of the web ;-)